src/transformers/models/clip/configuration_clip.py

```diff
@@ -16,8 +16,14 @@
 
 import copy
 import os
-from typing import Union
+from collections import OrderedDict
 
+from typing import Any, Mapping, Union, Optional
+
+from transformers import TensorType
+from transformers.processing_utils import ProcessorMixin
```

Comment:
> I took a deeper look @unography! There are a few things to correct regarding imports and type hints to make tests pass. I also have comments about the changes to the forward method of the model.
> 
> Besides, which version of black have you used to format the code? Did you use the command `make style` to do it? Because there are many files unrelated to this PR that have been reformatted and this should not have happened.

my black version is `22.6.0`, I think I did `make fix-copies` but tests were still failing so I did `black .` from inside the CLIP folder.
what is the correct way to fix this?

Reply:
Try with black 22.3, and use the command `make style` from the root of the repo to format the code.